src/libs/actions/Policy.js

```diff
@@ -95,6 +95,8 @@ function getPolicyList() {
                     // And overwrite them with only the ones returned by the API call
                     ...policyDataToStore,
                 });
+
+                Onyx.set(ONYXKEYS.ARE_ALL_POLICIES_LOADED, true);
```

Comment:
I'm trying to follow this, and I think I understand the reasoning... however, there is just something about it that doesn't make sense. Isn't this the same thing as subscribing to `ONYXKEYS.COLLECTION.POLICY` and when it's not empty, then you know the policies are loaded?

Reply:
Or maybe just when it's `!null`. There is probably a case where it's OK for the collection to be an empty array.

Reply:
Ohh I didn't consider doing it by checking null.

I was thinking the natural way would be to use `policies`but it would default to `[]` because of defaultProps overrides null values (and I'd assume policies should be default `[]`) or not work for users who just don't have policies yet.

But maybe I can make the `prop` in `withOnyx` `arePoliciesLoaded`, and instead just check for whether the `ONYXKEYS.COLLECTION.POLICY` key is `!null` in the `withOnyx` wrapper. That might not make sense how I described it, but I'm going to code it out and it might be more clear. I'll get rid of this key if that works fine.